File: src/main/java/org/pl/parser/ast/StatementListNode.java
package org.pl.parser.ast;

import java.util.List;

public class StatementListNode implements INode {
    public final List<INode> statements;

    public StatementListNode(List<INode> statements) {
        this.statements = statements;
    }

    @Override
    public String toString() {
        return "StatementListNode{" +
                "statements=" + statements +
                '}';
    }
}

File: src/main/kotlin/org/leaflang/parser/ast/StatementListNode.kt
package org.leaflang.parser.ast

import org.leaflang.parser.utils.NodePosition

/**
 * A statement list is a list of statement where each of them is
 * interpreted independently.
 */
class StatementListNode(override val position: NodePosition,
                        val statements: List<INode>) : INode {
    override fun toString() = "StatementListNode(statements=$statements)"
}